vendor.conf

```diff
@@ -1,7 +1,7 @@
 github.com/agl/ed25519 d2b94fd789ea21d12fac1a4443dd3a3f79cda72c
 github.com/Azure/go-ansiterm d6e3b3328b783f23731bc4d058875b0371ff8109
 github.com/containerd/continuity 35d55c5e8dd23b32037d56cf97174aff3efdfa83
-github.com/coreos/etcd v3.2.1
+# github.com/coreos/etcd v3.2.1
```

Comment:
you can remove comments

Reply:
Yes please, also would be good to keep this file sorted. Right now the diff is hard to read because some deps moved